src/output/json.cpp

```diff
@@ -0,0 +1,461 @@
+#include <iomanip>
+#include <string>
+
+#include "output/json.h"
```

Comment:
Translating data structures to JSON output seems like something there should be a standard library for, but maybe I have been coddled by Go's JSON handling features. Is constructing JSON strings by hand the easiest way here?

Reply:
Makes sense, thanks.